src/middleware/router.js

```diff
@@ -441,6 +452,12 @@ function sendHttpRequest (ctx, route, options) {
       method = https
     }
 
+    /*
+     This is used to determine whether the secondary route should finish execution.
+     It should not when there is a failure on the primary route
```

Comment:
Reading this comment on its own seems a bit ambiguous in terms of what its trying to address